conbench/metrics.py

```diff
@@ -190,13 +190,14 @@ def periodically_set_q_rem() -> None:
     """
 
     def func():
+        log.info("periodically_set_q_rem(): initiate")
         while True:
-            time.sleep(10)
+            time.sleep(1)
```

Comment:
This probably does not matter at all but is a more reasonable order of magnitude to pick.